frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/common.unit.spec.tsx

```diff
@@ -17,24 +17,23 @@ describe("SettingsEditor", () => {
       });
 
       await userEvent.click(screen.getByText("Embedding"));
-      await userEvent.click(screen.getByText("Interactive embedding"));
-      expect(screen.queryByText("Authorized origins")).not.toBeInTheDocument();
+      expect(screen.getByText("Interactive embedding")).toBeInTheDocument();
       expect(
-        screen.queryByText("SameSite cookie setting"),
-      ).not.toBeInTheDocument();
+        screen.getByText(/Embed dashboards, questions/),
+      ).toBeInTheDocument();
     });
 
-    it("should allow visiting the full-app embedding page even if embedding is not enabled", async () => {
+    it("should redirect users back to embedding settings page when visiting the full-app embedding page when embedding is not enabled", async () => {
```

Comment:
Fix the test name to match the new behavior (that has been changed for quite a while)